packages/generator/src/generator.mts

```diff
@@ -99,12 +99,11 @@ const debounce = (callback: () => void) =>
 		++debounceCounter,
 	)
 
-export const startGenerator = async (config?: GeneratorConfig, watchFiles = true): Promise<void> => {
```

Comment:
was this function never used by passing a config?

Reply:
Yes, only `undefined`

Reply:
Ah, must have been an older pattern I got rid of while refactoring